spec/client_spec.js

```diff
@@ -574,6 +577,19 @@ describe('Client', function() {
         clock.restore();
         expect(promise).to.be.rejectedWith(Error, 'Invocation request timeout').and.notify(done);
       });
+
+      it('doesnt reject whitelisted promises after 5 seconds', function(done) {
+        var clock = sinon.useFakeTimers();
+        promise = subject.invoke('instances.create');
+        clock.tick(10000);
+        clock.restore();
+        expect(promise).to.eventually.become({ errors: {}, 'instances.create': { url: 'http://a.b' } }).and.notify(done);
+        window.addEventListener.callArgWith(1, {
+          origin: subject._origin,
+          source: subject._source,
+          data: { id: requestsCount, result: { errors: {}, 'instances.create': { url: 'http://a.b' } } }
+        });
+      });
```

Comment:
Nope, `wrappedPostMessage` is not exported and invoke will throw unless it's called with a single string argument.